docs/75 LeetCode Questions/Arrays&Strings/7. Product of Array Except Self.md

```diff
@@ -0,0 +1,54 @@
+---
+id: product-of-array-except-self-238
+title: Product of Array Except Self
+sidebar_label: Leetcode 238
+tags: [Leetcode, String, Array, DSA, Product of Array Except Self]
+description: Given an integer array nums, return an array answer such that answer[i] is equal to the product of all the elements of nums except nums[i].
+---
```

Comment:
this is part of the props for our setup so, you need to use `key - value` pair and value as a string only and not use any programming keywords